bin/install

```diff
@@ -19,7 +19,7 @@ curl --silent --show-error --fail --location \
 
 # install packages.txt
 curl --silent --show-error --fail --location \
-    'https://raw.githubusercontent.com/fs/atom-bootstrap/master/config/packages.txt' \
+    'https://raw.githubusercontent.com/fs/atom-bootstrap/master/config/packages' \
```

Comment:
What is the reason to name it `config/packages` but save it locally as `packges.txt`?